schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -10,21 +10,41 @@ vars:
     SOFTFAIL_BSC1063638: 1
     VALIDATE_ETC_HOSTS: 1
 schedule:
+    - "{{pre_boot_to_desktop}}"
     - boot/boot_to_desktop
     - console/prepare_test_data
     - console/consoletest_setup
     - console/hostname
-    - yast2_gui/yast2_control_center
```

Comment:
this test was first to check first if all the modules are available, I have doubt about conditional schedule for this particular test suite due to gui test are quite unstable, feel free to create separate files if you see strange results.
I also noticed that you separated those yast2_lan_restart_* which makes sense to see them together. If you still want to keep one file you could just include all modules in {{yast2_modules}} and repeat them for each block. Also helps to understand that variable name, because now there are some modules out and some modules in.

Reply:
> I also noticed that you separated those yast2_lan_restart_* which makes sense to see them together. 

I separated them in a way that they are together xD . I mean that they run consecutively, one after the other.  I am under the impression that we will extend the coverage later, so these conditions will change. Putting all YaST modules in the conditional schedule would make it more clear and more flexible but in that case, there will be a lot of duplication. So what is best?

Reply:
I think will save time in the future thinking why some particular order doesn't work. I would go with the safest way, or duplication or several files, is up to you.

Reply:
Currently, the schedule looks very simple as ppc and aarch run just one module separate than the main schedule. I will leave it as it is and in the future, we can modify it.

Reply:
for me it is also fine if it works and is stable in the order that you currently have, my only concert is with `yast2_gui/yast2_control_center` which could deserve another extra condition to schedule it the first one only for x86_64, so we could catch some module which is not even able to be opened, wdyt?

Reply:
I changed it so that all yast modules are in the conditional schedule.

Reply:
still remains simple, thanks for the change.